include/integrations/smtp/settings.php

```diff
@@ -86,6 +86,23 @@
 	);
 	echo json_encode($response);
 } else {
+
+	$smtp_settings_mode = $_REQUEST['smtp_settings'];
+	/**
+	 * delete Incoming Mail Server Configuration
+	 */
+	if ($smtp_settings_mode == 'inc_set') {
+		$smtpconfig->clearInc_SMTPSettings();
+		header("Location: index.php?action=integration&module=Utilities&_op=getconfigsmtp&savemode=false");
```

Comment:
I think this is not needed. If you are in this "else" it is already getconfigsmtp, so you can just let it continue. Please delete it and try it to make sure it works the same with the redirect and without it. (in fact, I think it isn't working already, the menu HTML is breaking this header but you don't see the message because you don't have error reporting on, could be?)